projects/ngx-formentry/src/components/workspace-launcher/workspace-launcher.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'ofe-workspace-launcher',
+  templateUrl: './workspace-launcher.component.html'
+})
+export class WorkspaceLauncherComponent {
+  @Input() public id: string;
+  @Input() public buttonLabel: string;
+  @Input() public workspaceName: string;
+
+  public handleClick() {
+    if (!window['_openmrs_esm_patient_common_lib']) {
```

Comment:
Should these checks go somewhere where they will run at form load time, rather than when clicked? `onInit` or something?

Reply:
Maybe do a one-time check the first time `toWorkspaceLauncher()` is called; that way, we're still only checking on forms that are using this component, but the check is done during the render cycle?

Reply:
Cool, did that.